staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -47,8 +47,8 @@ var (
 			Name:      "transformation_duration_seconds",
 			Help:      "Latencies in seconds of value transformation operations.",
 			// In-process transformations (ex. AES CBC) complete on the order of 20 microseconds. However, when
-			// external KMS is involved latencies may climb into milliseconds.
-			Buckets:        metrics.ExponentialBuckets(5e-6, 2, 14),
+			// external KMS is involved latencies may climb into hundreds of milliseconds.
+			Buckets:        metrics.ExponentialBuckets(1e-5, 2, 20),
```

Comment:
I don't think you should increase the smallest bucket, and I think the biggest bucket should be close to 60 seconds. It might be OK for the factor to be a bit larger than 2.

Reply:
the comment said the in-process transformations complete in the order of 20 microseconds = 2e-5. setting it as 1e-5 seems ok.

done.

Reply:
I can't remember if I did the math wrong or was thinking that 0, 10, 20 microseconds was not enough resolution :)

Either way, seems fine to leave the bucket small.